### DISCUSSION TITLE
DKG Cards that show most KG, DG, DK overlap

### DISCUSSION THREAD

**Discussion by joe-davenport** [MKT-Playbook/Hub#21] [Feature requests]
When we have a favorite D,K,G it would be nice to see which associated items in the other categories have the most top shelf overlap. Building sets around your favorite items would be easier if I could see the most common top shelf items to group DKG for future.

**Comment by TitusPosiadly** [comment]
Hey, thanks for the request. Yeah, I want to start working on a better coverage tool as soon as I have a bit more time again. Hopefully, I can spare some time between the holidays. I will also add a section for favorite cards in there somehow.

**Comment by joe-davenport** [comment]
Someone recently made the graphic of the kart and glider pairing for each of the drivers not long ago, but it would be nice to know the technical pairing for each driver to see their most favored kart and glider based on how often they are duplicated on top shelf.

I’m picturing this at the card level for each item, then two icons of the favorites, possibly also having that drop down area like the cups have to show the alts or ties.

On another note, I’m thinking the max levels of giant bananas are now the top item and might be under counted, so I’m heading to to yellow mii next unless a holiday coin box mii.

On Dec 7, 2022, at 10:01 AM, Titus Posiadly ***@***.***> wrote:

﻿

Hey, thanks for the request. Yeah, I want to start working on a better coverage tool as soon as I have a bit more time again. Hopefully, I can spare some time between the holidays. I will also add a section for favorite cards in there somehow.

**Reply by TitusPosiadly** [reply]
Gotchu. Added it to the list. The problem with giant bananas is that the potential actions differ very much depending on the track. I know this is the same for coinbox but because the app calculated some ridiculously high scores on some tracks for giant banana I decided to nerf giant banana until I figure out a better way to check this. But I also still think giant banana is extremely good since the update and can get very close to if not even get better scores then coinboxers

**Comment by joe-davenport** [comment]
Yes, the narrow tracks where the bots can not escape is the only time I can exceed your 100% estimate on score. It’s so good that I also feel like Nintendo will take away some the hits, because it seems the bots don’t have the typical invincibility after a hit which would wipe out more bananas without earning points. Or they may end up capping the max bananas on the track to something lower.

On Dec 11, 2022, at 7:24 AM, Titus Posiadly ***@***.***> wrote:

﻿

Gotchu. Added it to the list. The problem with giant bananas is that the potential actions differ very much depending on the track. I know this is the same for coinbox but because the app calculated some ridiculously high scores on some tracks for giant banana I decided to nerf giant banana until I figure out a better way to check this. But I also still think giant banana is extremely good since the update and can get very close to if not even get better scores then coinboxers

**Reply by TitusPosiadly** [reply]
Yeah, I'm pretty sure they already have a cap on the max number of items possible laying on the track. Don't know how high that is though.

**Comment by joe-davenport** [comment]
I now remember also exceeding with triple coin/box frenzy, but also very hard to get a double/triple banana frenzy, so it doesn’t really make sense to have anyone trying to hit that score

On Dec 11, 2022, at 11:43 AM, Joe Davenport ***@***.***> wrote:

﻿
Yes, the narrow tracks where the bots can not escape is the only time I can exceed your 100% estimate on score. It’s so good that I also feel like Nintendo will take away some the hits, because it seems the bots don’t have the typical invincibility after a hit which would wipe out more bananas without earning points. Or they may end up capping the max bananas on the track to something lower.

On Dec 11, 2022, at 7:24 AM, Titus Posiadly ***@***.***> wrote:

﻿

Gotchu. Added it to the list. The problem with giant bananas is that the potential actions differ very much depending on the track. I know this is the same for coinbox but because the app calculated some ridiculously high scores on some tracks for giant banana I decided to nerf giant banana until I figure out a better way to check this. But I also still think giant banana is extremely good since the update and can get very close to if not even get better scores then coinboxers
